user_tools/custom_speedup_factors/validate_qualification_estimates.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# -----
+"""Spark RAPIDS speedup factor validation script"""
+
+import argparse
+import os
+import glob
+import subprocess
+
+import pandas as pd
+from tabulate import tabulate
+
+parser = argparse.ArgumentParser(description="Speedup Factor Validation")
```

Comment:
Should we add document of how to run this? I am fine if it's going to be a follow on PR.

Reply:
Yes, I'll add instructions in the README on how to run it as part of this PR.